src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+.product-card {
+  justify-content: center;
```

Comment:
The `.product-card` block should not use `justify-content` as it does not use `display: flex` or `display: grid`. This property will have no effect on the layout.